Content.Server/Interfaces/IRadio.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Robust.Shared.Interfaces.GameObjects;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.Interfaces
+{
+    public interface IRadio
+    {
+        void Receiver(string message, int channel, IEntity speaker);
+
+        void Broadcast(string message, IEntity speaker);
+
+        List<int> GetChannels();
```

Comment:
You could use a read-only property for Channels i.e.
public IReadOnlyCollection<int> Channels => _channels

Also look at making the int something like a ushort (minor difference).